.github/workflows/sdk-jobs-automl-standalone-jobs-automl-forecasting-task-energy-demand-automl-forecasting-task-energy-demand-advanced-mlflow.yml

```diff
@@ -1,101 +0,0 @@
-# This code is autogenerated.
```

Comment:
Looks like this file is autogenerated? Is it expected for this to be deleted?

Reply:
@novaturient95 As this file no longer exists, it was deleted. Is this the way or do we need to run a workflow to ensure deletion